chain/chain/src/chain_update.rs

```diff
@@ -171,7 +171,7 @@ impl<'a> ChainUpdate<'a> {
         let height = block.header().height();
         match resharding_results {
             ReshardingResults::ApplyReshardingResults(mut results) => {
-                tracing::debug!(target: "resharding", height, ?shard_uid, "process_resharding_results apply");
+                tracing::debug!(target: "resharding", height, shard_id=?shard_uid.shard_id(), "process_resharding_results apply");
```

Comment:
Why was this change necessary? I thought we are prioritizing use of `shard_uid` to avoid ambiguities?

Reply:
For resharding we should indeed use shard_uid. I'm guessing Tayfun wants the shard_id for consistent analysis, but we can export both fields.

Reply:
> we can export both fields

Conversely traces are measured by byte. Its fine to do this if the event is rare enough, but for frequently triggered output we should stay mindful of not duplicating information unnecessarily.

Reply:
Ok, let me revert that back to shard uid and we can handle it in the tracing collector/analyzer side. no need to add extra field with the same info.